### PR TITLE
Add validator docker-compose

### PR DESCRIPTION
**Pull request by vladupshot**
Add IaC and man to run a validator with docker-compose

### CODE REVIEW

**Comment by relyt29** [comment]
#49 makes use of init.sh i don't love deleting it

File: README.md [hunk 1]
```diff
@@ -29,3 +29,128 @@ make init
...
+    "update_time": "2024-02-26T22:50:31.187119394Z"
+  },
+  "min_self_delegation": "1"
+}
+```
+### Check the voting power of your validator node
+*NOTE:* please allow 30-60 seconds for the output to be updated
+
+`allorad --home=$APP_HOME status | jq -r '.validator_info.voting_power'`
+- Output should be > 0
```
Comment (guilherme-brandao) [inline README.md:144]: Not sure if this is needed rn since we don't have a gov module yet

**Review by guilherme-brandao — COMMENTED** [review]
Lgtm! Need to rebase before merging

**Review by guilherme-brandao — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (vladupshot) [reply README.md:144]: thios is a readonly, just to check if it staked successfully

and it works, so why not)))

**Pull request merged**